modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -364,6 +374,45 @@ private String _getObjectEntryPermissionName(long objectDefinitionId) {
 		return ObjectDefinition.class.getName() + "#" + objectDefinitionId;
 	}
 
+	private void _hasPreventDeletionType(long objectDefinitionId) {
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipLocalService.getObjectRelationships(
+				objectDefinitionId);
+
+		_hasPreventTypeRelationship = false;
+
+		for (ObjectRelationship objectRelationship : objectRelationships) {
+			if (StringUtil.equals(
+					objectRelationship.getDeletionType(),
+					ObjectRelationshipConstants.DELETION_TYPE_PREVENT)) {
+
+				_preventObjectRelationships.add(objectRelationship);
+				_hasPreventTypeRelationship = true;
+			}
+		}
+	}
+
+	private boolean _hasRelatedModels(
```

Comment:
I think we could rename this method to something more similar to the name of the method we call inside (getOneToManyRelatedObjectEntriesCount). Like: _hasRelatedObjectEntries ? What do you think?

Reply:
I agree